### PR TITLE
Refactor power meter and flipmount drivers

### PR DESCRIPTION
**Pull request by WalterMwaniki**
- Minor updates to flipmount and powermeter device drivers.  
- Inherit base `VoxelDevice` class.

### CODE REVIEW

File: voxel/descriptors/deliminated_property.py [hunk 1]
```diff
@@ -26,7 +26,10 @@ def __set__(self, instance, value):
             raise AttributeError("can't set attribute")
         if self.step is not None:
             value = round(value / self.step) * self.step  # if step size, ensure value adheres to this
-        self._fset(instance, min(self.maximum, max(value, self.minimum)))  # if under/over min/max, correct
+        # if minimum/maximum are callable, call them with instance
+        maximum = self.maximum(instance) if callable(self.maximum) else self.maximum
+        minimum = self.minimum(instance) if callable(self.minimum) else self.minimum
+        self._fset(instance, min(maximum, max(value, minimum)))  # if under/over min/max, correct
```
Comment (micahwoodard) [inline voxel/descriptors/deliminated_property.py:32]: That's cool I like that implementation

File: voxel/devices/base.py [hunk 2]
```diff
@@ -2,15 +2,22 @@
...
 import logging
 
+
 class VoxelDevice(ABC):
     """Base class for all voxel devices."""
 
     def __init__(self, id: str):
         self.id = id
-        self.log = logging.getLogger(f'{__name__}.{self.__class__.__name__}')
+        self.log = logging.getLogger(f"{self.__class__.__name__}[{self.id}]")
```
Comment (micahwoodard) [inline voxel/devices/base.py:11]: Why did you change the format here?

File: voxel/devices/lasers/simulated.py [hunk 3]
```diff
@@ -37,31 +40,45 @@ def percentage_split(self, value):
...
-        self.ser = Serial
-        self._simulated_power_setpoint_m = 10.0
+        self._combiner = combiner
+        self._simulated_power_setpoint_mw = 10.0
         self._max_power_mw = 100.0
         self._modulation_mode = 'digital'
         self._temperature = 20.0
         self._cdrh = 'ON'
+        self._status = []
+
```
Comment (micahwoodard) [inline voxel/devices/lasers/simulated.py:64]: I think we should remove the combiner and a keyword since it's not really doing anything and it's a bit misleading for a user to create a SimulatedCombiner and have it do nothing

**Comment by micahwoodard** [comment]
Also @WalterMwaniki could you go through and either delete or update the test files that reference the obis and oxxius lasers? Those are pretty out of date so it might be worth updating them

**Comment by adamkglaser** [comment]
@WalterMwaniki this one is OK to merge after addressing Micah's comments

File: voxel/devices/lasers/simulated.py (hunk 3 above)
Comment (WalterMwaniki) [reply voxel/devices/lasers/simulated.py:64]: Makes sense, I'll remove the combiner from the simulated laser driver.
